### ISSUE TITLE
[BUG] Lookup value returning lowercase strings

### ISSUE THREAD

**Issue by allanzh** [Microsoft/EasyRepro#851]
## Bug Report

**EasyRepro Version**
  - [ ] Microsoft Dynamics 365 Online Version 8.2 (8.2.x) (DB 8.2.x) online
  - [ ] Microsoft Dynamics 365 Online Version 9.0 (9.0.x) (DB 9.0.x) online
  - [ ] Microsoft Dynamics 365 Online Version 9.0.2 (9.0.2) (DB 9.0.2) online
  - [x] Microsoft Dynamics 365 Online Version 9.1 (9.1.x) (DB 9.1.x) online
  
**UCI or Classic Web**
  - [x] UCI (XrmApp)
  - [ ] Classic Web (XrmBrowser)

**Online or On Premise**
  - [x] Online
  - [ ] On Premise

**Browser**
  - [x] Chrome
  - [ ] Firefox
  - [ ] IE
  - [ ] Edge

**Describe the bug**  
When I try to get an Lookup value, GetValue function returns a string with lowercase letters.

**Code to reproduce**  

```csharp
string value = xrmApp.Entity.GetValue(new LookupItem { Name = "FieldName"});
Assert.AreEqual("SomeText", value);
```

**Expected behavior**  

In order to do a better comparison between strings, GetValue function needs to return the exact string that is showing in lookup fields.

**Screenshots**  

![wrong comparison](https://user-images.githubusercontent.com/2539476/81021028-92ca3d00-8e40-11ea-8667-1a7369747ae6.png)

**Additional context**  

In my case I'm doing an workarround using .ToLower() string function, but its not the right way to solve it.

**Comment by TYLEROL** [comment]
@allanzh - Thanks for raising this. I just tried to replicate this issue, but I am not reproducing the problem with the **develop** branch. There have been an array of changes made recently that are not yet merged with the NuGet packages. I plan to merge them either later this week, or ~approx. end of next week.

_**Removed ask**_

**Edit**: Disregard my prior request, this was fixed in this PR - #772 . It will be fixed when changes are merged to NuGet and the releases/v9.1 branch. This should happen by end of next week at the latest.

Warm Regards,
Tyler
